Plandomizer.py

```diff
@@ -572,7 +572,12 @@ def fill(self, window, worlds, location_pools, item_pools):
 
                 # Update item_pool
                 if item is not None:
-                    if item not in self.item_pool:
+                    item_in_pool = False
+                    for record in self.item_pool:
```

Comment:
This entire loop can just be `item_in_pool = item.name in self.item_pool`, right? So the original if statement almost works but would be `if item.name not in self.item_pool`...